src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> set = new TreeSet<>();
+        try (FileInputStream fin = new FileInputStream(fileName)) {
+            int i = 0;
+            while ((i = fin.read()) != -1) {
+                if (Character.isLetter((char) i)) {
+                    set.add(Character.toLowerCase((char) i));
+                }
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException();
```

Comment:
- exceptions should be informative
- you should better rethrow a RuntimeException if you think that this edge case should never occur